src/popup/router/components/Header.vue

```diff
@@ -88,91 +98,105 @@ export default {
 
 <style lang="scss" scoped>
 @import '../../../styles/variables';
+@import '../../../styles/typography';
```

Comment:
In this case we can remove `variables` import as it is already imported in `typography`

Reply:
[`@import`](https://sass-lang.com/documentation/at-rules/import) is deprecated, let's use [`@use`](https://sass-lang.com/documentation/at-rules/use) instead. `@use` working differently and probably we won't be needed to remove "duplicate" imports.

Reply:
Using `@use` instead of `@import` will require making some global changes first.

Reply:
Let's log and address this separately and not stall this PR because of this change.

Reply:
#832